libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -43,6 +45,7 @@ export class BookSearchComponent {
 
   addBookToReadingList(book: Book) {
     this.store.dispatch(addToReadingList({ book }));
+    this.showSnackBar(book);
```

Comment:
Here you are just showing the snackbar at the same time that the `addToReadingList` action is dispatched. What if the action fails? I think you should show the snackbar after the `addToReadingList` action is confirmed!

Reply:
Updated code to display snackbar after confirmed 'addBookToReadingList' in reading-list.effects.ts file.